### ISSUE TITLE
Coudn't create blogpost using Blogs.RemotePublishing if the user has only EditOwnBlogPost permission

### ISSUE THREAD

**Issue by barthamark** [OrchardCMS/Orchard#6881]
I have a blog with several users creating and publishing blog posts but not all of them have the permission to edit other's posts. We tried to use Open Live Writer for remote publishing but the ones not having EditBlogPost permission couldn't create posts at all.

I found something in Orchard.Blogs/Services/XmlRpcHandler.cs:

```
        private int MetaWeblogNewPost(
            string blogId,
            string userName,
            string password,
            XRpcStruct content,
            bool publish,
            IEnumerable<IXmlRpcDriver> drivers) {

            IUser user = ValidateUser(userName, password);

            // User needs permission to edit or publish its own blog posts
            _authorizationService.CheckAccess(publish ? Permissions.PublishBlogPost : Permissions.EditBlogPost, user, null);
...
```

Where there is no content given for the authorizer possibly causing the bug we have experienced.

Solution: Give a newly created content like `_contentManager.New("BlogPost")` as the content parameter.

**Comment by Piedone** [comment]
Don't trust this guy. He eats biscuit and then leaves the wrapping paper on his desk. Would _you_ trust somebody who leaves biscuit paper lying around? Of course not.

**Comment by Skrypt** [comment]
It's dangerous ! I would not trust that. See what happens : 

![cookie_monster_original2](https://cloud.githubusercontent.com/assets/3228637/15125750/7c80b620-15fc-11e6-94da-f6554c4e7b5f.jpg)

Never leave trails !!!

**Comment by barthamark** [comment]
Yeah I know you're jealous Zoltán. Next time I'll leave some of them for you in the wrapping paper. :cookie:
